src/lib/y2users/linux/reader.rb

```diff
@@ -20,11 +20,93 @@
 require "yast2/execute"
 require "y2users/linux/base_reader"
 
+Yast.import "MailAliases"
+
 module Y2Users
   module Linux
     # Reads users configuration from the system using `getent` command.
     class Reader < BaseReader
-    private # rubocop:disable Layout/IndentationWidth
+      # @see BaseReader#read
+      # @note some of these #read_* methods could be moved to the
+      #   {BaseReader#read} once they allow reading from a specific location
+      #   (i.e., compatible with {LocalReader} too)
+      def read
+        config = super
+
+        read_root_aliases(config)
+        read_home_permissions(config)
+        read_authorized_keys(config)
+        read_useradd_config(config)
+        read_login(config)
+
+        config
+      end
+
+    private
+
+      # Set root aliases (i.e., users receiving system mails)
+      #
+      # @see Yast::MailAliases#GetRootAlias
+      #
+      # @param config [Config]
+      def read_root_aliases(config)
+        Yast::MailAliases.GetRootAlias.split(", ").each do |name|
+          user = config.users.by_name(name)
+          user.receive_system_mail = true if user
+        end
+      end
+
+      # Command for reading home directory permissions
+      STAT = "/usr/bin/stat".freeze
+      private_constant :STAT
+
+      # Reads home permissions
+      #
+      # @return [Array<Y2Users::User>]
+      def read_home_permissions(config)
+        config.users.reject(&:system?).each do |user|
+          next unless user.home && Dir.exist?(user.home.path)
+
+          user.home.permissions = Yast::Execute.on_target!(
+            STAT, "--printf", "%a", user.home.path,
+            stdout: :capture
+          ).prepend("0")
```

Comment:
Please, add a comment to explain the result of start %a and why it is needed to prepend a 0.

Reply:
While trying to write down _why it is needed_ I realized that using the [`#` printf flag character](https://www.man7.org/linux/man-pages/man3/printf.3.html) it's better. Honestly, [`stat`documentation](https://github.com/coreutils/coreutils/blob/a0468760ffa9f7843bbd0d6cf60685f63d33b600/src/stat.c#L1783) was not so clear to me the first time I read it.

Please, check the added comment and suggest changes if you find it confusing.

Thanks!